context/app/static/js/pages/Dataset/Dataset.jsx

```diff
@@ -65,10 +65,13 @@ function DatasetDetail(props) {
     last_modified_timestamp,
     description,
     status,
+    sub_status,
     mapped_data_access_level,
   } = assayMetadata;
   const isLatest = !('next_revision_uuid' in assayMetadata);
 
+  const combinedStatus = sub_status || status;
```

Comment:
`sub_status` always takes precedence over status? Could there be a situation where the `status` is more severe and worth attention?

Reply:
Maybe? Eventually? but right now the only usage is retracted.